plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -248,6 +248,26 @@ private function add_general_group_blocks() {
 				],
 			]
 		);
+		// Downloads section.
+		$general_group->add_section(
+			[
+				'id'         => 'product-downloads-section',
+				'order'      => 40,
+				'attributes' => [
+					'title'       => __( 'Downloads', 'woocommerce' ),
+					'description' => __( "Add any files you'd like to make available for the customer to download after purchasing, such as instructions or warranty info.", 'woocommerce' ),
+				],
+			]
+		)->add_block(
+			[
+				'id'         => 'product-downloads',
+				'blockName'  => 'woocommerce/product-downloads-field',
+				'order'      => 10,
+				'attributes' => [
+					'images' => [],
```

Comment:
Should this be `downloads`?

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/40526/commits/82381641356554dabd2c8794593f35ec28857997